cluster-management/setup/configuring-date-and-timezone/index.md

```diff
@@ -8,7 +8,11 @@ weight: 7
 
 # Configuring Date and Timezone
 
-NTP is used to to keep clocks in sync across machines in a CoreOS cluster. The ntpd service is responsible for keeping each machines local clock in sync with a configured set of time servers. The services will automatically start by default. To check if the ntpd service is running, run the follow command:
+NTP is used to to keep clocks in sync across machines in a CoreOS cluster.
+CoreOS [681.0.0][681.0.0] uses [systemd-timesyncd][systemd-timesyncd] as the
+default NTP client, prior to that [ntpd][ntp.org] was used. Depending on the
+your version of CoreOS one of the two services will automatically start. To
+check if the ntpd service is running, run the follow command:
 
 ```
 systemctl status ntpd
```

Comment:
If your machine is using `systemd-timesyncd`, do you still ask for the `ntpd` unit?